benchmark/http/chunked.js

```diff
@@ -11,14 +11,14 @@
 var common = require('../common.js');
 
 var bench = common.createBenchmark(main, {
-  num: [1, 4, 8, 16],
-  size: [1, 64, 256],
+  n: [1, 4, 8, 16],
+  len: [1, 64, 256],
```

Comment:
So test-benchmark-http.js can set the parameters for it (otherwise it will need to add `--set num=1 --set size=1`...also I think it's better to keep the configuration names consistent)